bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties

```diff
@@ -11,8 +11,8 @@ bridge-type.velux.klf200.description = The Velux KLF200 represents a gateway to
 #
 # Thing types descriptions
 #
-thing-type.velux.binding.label = Velux Binding Information
-thing-type.velux.binding.description = Presents the status of the openHAB Velux binding.
+thing-type.velux.addon.label = Velux Binding Information
```

Comment:
I think there is potentially an issue with this binding information thing. Probably the thing.xml needs to be changed as well?

Reply:
No, that one should not have been amended. `binding` is the thing name in this case and that's fine (besides that IMO a thing that modifies the binding behavior should never have made it into the code-base, it's totally anti-pattern).

Reply:
> a thing that modifies the binding behavior should never have made it into the code-base

Agreed. (Not my code). But to be fair, it does not modify any behavior; it simply provides some static text information; but if I would write the binding myself, that information could be in a bridge property, rather than a biding thing..